nextjs/fjquant-2458/pages/index.js

```diff
@@ -0,0 +1,94 @@
+import { useEffect, useState } from "react";
+import Layout from "@/components/Layout";
+import Card from "@/components/card";
+import styles from "../styles/Home.module.css"
+
+const Dogs = () => {
+
+  const [dogs, setDogs] = useState([]);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const dogs = await fetch("/api/dogs")
+          .then((res) => res.json())
+        setDogs(dogs)
+      } catch (err) {
+        console.log(err);
+      }
+    }
+    fetchData()
+  }, [])
+
+  return (
+    <>
+      <Layout pagina='inicio'>
+        <div>
+          <h1 className={styles.Title}> 10 random dogs Images</h1>
+            <div className={styles.container}>
+              {dogs.slice(0, 5).map((dog, index) => {
+                return <Card key={index} title={dog.title} img={dog.img} />
+              })}</div>
+              <div className={styles.container}>
+              {dogs.slice(5, 10).map((dog, index) => {
+                return <Card key={index} title={dog.title} img={dog.img} />
+              })}
```

Comment:
Aqui podrias simplificar el codigo usando solo un ciclo map no veo muy necesario tener dos, luego puedes acomodar bien las cards usando css y flex, ten cuidado que con dos contenedores exactamente iguales esta arrojando un warning en consola.

Reply:
Utilicé dos veces el map para lograr tener las dos filas con diferentes razas de perros. Con un solo map de 10 razas no lograba colocarlo en dos filas.

Reply:
podrias agregar a tus estilos en css lo siguiente, asi puedes controlar cuantos items por fila quieres mostrar:
`.entrada {
  flex-grow: 1;
  width: 20%;
}`
asi puedes ya podrias simplificar tu codigo
```suggestion
            <div className={styles.container}>
              {dogs.slice(0, 10).map((dog, index) => {
                return <Card key={index} title={dog.title} img={dog.img} />
              })}
```

Reply:
Excelente ayuda, ya lo ajusté. Gracias